.github/workflows/integration-test.yml

```diff
@@ -0,0 +1,21 @@
+name : Integration Tests
+on : 
+  schedule:
+  - cron: '30 4 * * 1'    
+jobs:
+  test:
+    runs-on: ubuntu-latest
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+    
+    container: 
+      image: pecan/depends:R4.1
```

Comment:
You need to either change this to an image that contains the current full pecan install or you have to install the latest version of pecan. The pecan/depends container just contains all the libraries and R packages that pecan needs to build, not pecan itself. Check with @robkooper but I think you want to use pecan/base or pecan/executor (Rob, note that none of these tests need the models themselves, just the pecan libraries)

Reply:
Do you think that, we should create a phony object in the makefile to only install base and modules and skip the models ? The current make command will install all the 3 during the workflow.